src/PharoLauncher-CLI-Tests/ConsoleListFormatterTest.class.st

```diff
@@ -162,64 +181,58 @@ ConsoleListFormatterTest >> testPrintLineSized [
 ]
 
 { #category : #test }
-ConsoleListFormatterTest >> testPrintList [
+ConsoleListFormatterTest >> testPrintListAsSton [
 
-	| stonString tableModeString |
-	formatter attributeLabels: #('Value' 'Squared' 'Raised to 3').
-	formatter attributeValueBlocks: { [:a |a asWords  ]. [:a |a squared asWords  ]. [:a |(a **3) asWords  ] }.
-	formatter domainObjects: #(1 2 3).
-	
-	"test if table is printed, if ston mode is disabled"
-	formatter printList.
-	tableModeString  := formatter outStream contents.
-	command resetOutStream.
-	formatter printListAttributes.
-	self assert: formatter outStream contents equals: tableModeString.
+	| domainObjects deserialized |
+	domainObjects := { PhLImageMock atRandom. PhLImageMock atRandom. PhLImageMock atRandom }.
+	formatter domainObjects: domainObjects.
 
-	"test for ston output flag"
-	command resetOutStream.
-	formatter outputAsSton: true.
-	formatter printList.
-	stonString  := formatter outStream contents.
-	command resetOutStream.
 	formatter printListAsSton.
-	self assert: formatter outStream contents equals: stonString.
+	deserialized := STON fromString: formatter outStream contents.
+	
+	self assert: deserialized size equals: 3.
+	self assert: deserialized first class equals: PhLImageMock.
 ]
 
 { #category : #test }
-ConsoleListFormatterTest >> testPrintListAsSton [
-
-	formatter domainObjects: {PhLImagesMock atRandom. PhLImagesMock atRandom. PhLImagesMock atRandom.}.
-	formatter printListAsSton.
+ConsoleListFormatterTest >> testPrintListShouldPrintAttributesTableWhenStonAndRowModesDisabled [
+	| tableModeString |
+	formatter attributeLabels: #('Value' 'Squared' 'Raised to 3').
+	formatter
+		attributeValueBlocks:
+			{[ :a | a asWords ].
+			[ :a | a squared asWords ].
+			[ :a | (a ** 3) asWords ]};
+		domainObjects: #(1 2 3);
+		printAttributeTable.
+	tableModeString := formatter outStream contents.
+	command resetOutStream.
```

Comment:
I'm not sure now, if resetting outstream before getting contents of outstream isn't even necessary in previous test case (e.g. when same test is evaluated twice?)